test/per_file_coverage.sh

```diff
@@ -4,8 +4,10 @@
 # for existing directories with low coverage.
 declare -a KNOWN_LOW_COVERAGE=(
 "source/common:95.9" # Raise when QUIC coverage goes up
+"source/common:96.5"
```

Comment:
Sorry for noticing this late, but I think these two can be combined in some subsequent PR.

Reply:
nice catch - fixed over in https://github.com/envoyproxy/envoy/pull/19787/files